runtime/lua/vim/lsp.lua

```diff
@@ -2215,12 +2215,12 @@ end
 ---
 ---@param bufnr (integer) Buffer handle, or 0 for current.
 ---@param method (string) LSP method name
----@param params (table|nil) Parameters to send to the server
----@param timeout_ms (integer|nil) Maximum time in milliseconds to wait for a
+---@param params? (table) Parameters to send to the server
+---@param timeout_ms? (integer) Maximum time in milliseconds to wait for a
 ---                               result. Defaults to 1000
 ---
----@return table<integer, {err: lsp.ResponseError, result: any}>|nil (table) result Map of client_id:request_result.
```

Comment:
I removed `(table)` because I think it was a hack because the full type wasn't included in the docs. Now the full type is generated, and the `(table)` isn't necessary.